prow/config/config.go

```diff
@@ -18,11 +18,11 @@ limitations under the License.
 package config
 
 import (
+	"bufio"
```

Comment:
I read this as `buffalo` and thought we were about to start using the best stdlib library in existence ...